seshdash/utils/model_tools.py

```diff
@@ -69,3 +69,46 @@ def get_measurement_verbose_name(measurement):
 
 def get_measurement_unit(measurement):
     return Daily_Data_Point.UNITS_DICTIONARY[measurement]
+
+
+def get_model_field_names(model):
+    """ 
+    This function returns the field names of fields
+    in a given model
+    """
+    fields = model._meta.get_fields()
+
+    field_arr = []
+
+    for field in fields:
+        field_arr.append(field.name)
+
+    return field_arr
+
+
+def get_status_card_items(site):
+    """
+    Returns the list of items to be displayed in the status card
+    """
+    status_card = site.status_card
+
+    if not status_card:
```

Comment:
What happens when a new site is created? It should get it's status card created by default.